static/app/components/feedback/feedbackOnboarding/useLoadFeedbackOnboardingDoc.tsx

```diff
@@ -27,7 +27,9 @@ function useLoadFeedbackOnboardingDoc({
     platform?.type === 'framework'
       ? platform?.id === 'capacitor'
         ? `capacitor/capacitor`
-        : platform?.id.replace(`${platform.language}-`, `${platform.language}/`)
+        : platform?.id === 'dart'
+          ? `dart/dart`
+          : platform?.id.replace(`${platform.language}-`, `${platform.language}/`)
       : `${platform?.language}/${platform?.id}`;
```

Comment:
If this ternary gets any longer we should extract it into a function.

Reply:
will do! it will definitely get longer... see

https://github.com/getsentry/sentry/blob/8652855a4b3e9aa2d9d72b72dccee94368533f96/static/app/components/onboarding/gettingStartedDoc/sdkDocumentation.tsx#L65

Reply:
![8h4hjj](https://github.com/getsentry/sentry/assets/187460/ac7ce089-2561-4281-9f28-1e261edf89d9)